src/url-util.js

```diff
@@ -17,4 +17,3 @@ export function getUrl(
 
   return `${BASE_URL}/${path}?${queryString.substring(1)}`;
 }
```

Comment:
I thought Prettier was adding a newline at the end of all our source files. I don't particularly like that, but I google that and it seems there are arguments to do it.